### PR TITLE
Patched http2-1.6.3

### PR DESCRIPTION
**Pull request by ouromoros**

### CODE REVIEW

File: patches/http2-1.6.3.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+diff --git a/Network/HPACK/Huffman/ByteString.hs b/Network/HPACK/Huffman/ByteString.hs
+index 96d501f..9d29f0f 100644
+--- a/Network/HPACK/Huffman/ByteString.hs
++++ b/Network/HPACK/Huffman/ByteString.hs
+@@ -47,7 +47,7 @@ copy dst (PS fptr off len) = withForeignPtr fptr $ \ptr -> do
+     let beg = ptr `plusPtr` off
+     memcpy dst beg (fromIntegral len)
+ 
+-foreign import ccall unsafe "string.h memcpy" c_memcpy
++foreign import java unsafe "@static eta.base.c_memcpy" c_memcpy
```
Comment (rahulmutt) [inline patches/http2-1.6.3.patch]: Shouldn't this be `eta.base.Utils.c_memcpy`?

**Comment by rahulmutt** [comment]
@ouromoros Do you plan on finishing up this PR?

**Comment by ouromoros** [comment]
@rahulmutt Oh, I fixed the error you mentioned. I thought it's ready to merge?

**Comment by rahulmutt** [comment]
@ouromoros My bad! I didn't get any notification so I thought this was still pending. Merging.

**Pull request merged**